src/convertToObject.js

```diff
@@ -7,6 +7,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const stylesObject = {};
+
+  const lines = sourceString.split(';');
+
+  lines.forEach(function (line) {
+    const part = line.split(':');
+
+    if (part.length >= 2) {
+      stylesObject[part[0].trim()] = part[1].trim();
```

Comment:
There is a potential issue if a style declaration contains more than one colon ':' character. The split will create more than two parts, and the additional parts will be ignored. You should handle this case to ensure the full value is captured.